cms/envs/production.py

```diff
@@ -537,7 +537,6 @@ def get_env_setting(setting):
 
 plugin_settings.add_plugins(__name__, plugin_constants.ProjectType.CMS, plugin_constants.SettingsType.PRODUCTION)
 
-########################## Derive Any Derived Settings  #######################
+########################## Derive Any Derived Settings #######################
```

Comment:
remove this line

Reply:
Only a space was removed, this line is exactly the same.